commit b5403eca16581b0a9127a4c4052952d801270366
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Wed Aug 8 01:55:04 2018 +0200

    hurd: Add missing symbols for proper libc_get/setspecific
    
            * htl/Versions (__pthread_getspecific, __pthread_setspecific): Add
            symbols.

diff --git a/htl/Versions b/htl/Versions
index 6a63a1b8a1..c5a616da10 100644
--- a/htl/Versions
+++ b/htl/Versions
@@ -150,6 +150,8 @@ libpthread {
     __cthread_keycreate;
     __cthread_getspecific;
     __cthread_setspecific;
+    __pthread_getspecific;
+    __pthread_setspecific;
     __pthread_getattr_np;
     __pthread_attr_getstack;
   }
